dataset/csv2lesson.py

```diff
@@ -12,11 +12,12 @@
     category = os.path.splitext(filename)[0]
     newfile = open(os.path.join("vocab",os.path.splitext(filename)[0]+".md"), "w")
     newfile.write('''---
-    title: '''+ category +
-    '''
-    description: 
-    icon: /media/icons/''' + os.path.splitext(filename)[0] + '''.svg
-tags: {easy,vocab}
+title: '''+ category.capitalize() +
```

Comment:
capitalize is the job of CSS/rendering

Also use : https://realpython.com/python-string-formatting/#3-string-interpolation-f-strings-python-36